dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/imports/bundle/RelationshipImportTest.java

```diff
@@ -80,6 +82,17 @@ void successImportingRelationships() throws IOException {
     assertThat(importReport.getStats().getCreated(), is(2));
   }
 
+  @Test
+  void shouldFailWhenUserNotAuthorizedToCreateRelationship() throws IOException {
+    userA = userService.getUser("o1HMTIzBGo7");
+    TrackerImportParams params = TrackerImportParams.builder().userId(userA.getUid()).build();
+    ImportReport importReport =
+        trackerImportService.importTracker(params, fromJson("tracker/relationships.json"));
+    assertThat(importReport.getStatus(), is(Status.ERROR));
```

Comment:
that is already asserted by assertHasError

Reply:
`assertThat(importReport.getStatus(), is(Status.ERROR));` is already asserted by `assertHasError`. Can you please remove that line.

And add newline between setup, method under test and assertions. So it's easy to spot them.